frontend/ui/widget/bookmapwidget.lua

```diff
@@ -973,15 +1043,25 @@ function BookMapWidget:update()
                     if item.page == p_start then
                         cur_left_spacing = self.row_left_spacing + self.flat_toc_level_indent * (item.depth-1)
                         local txt_max_width = self.row_width - cur_left_spacing
-                        table.insert(self.vgroup, HorizontalGroup:new{
-                            HorizontalSpan:new{
-                                width = cur_left_spacing,
-                            },
+                        local toc_title = FrameContainer:new{
+                            margin = 0,
+                            padding = Size.border.thin * 3,
```

Comment:
I guess this adds some padding around the TOC title in flat mode, just so they can be highlighted.
I guess the nice handmade alignment I tried to achieve:
![image](https://github.com/user-attachments/assets/6389f3cb-f6ef-4e20-a37d-a1d07b3aeb71)
will then be lost as every TOC title will get more air on all sides. Not tested, but not happy with that if it happens.

Reply:
Fixed